src/main/groovy/com/twcable/grabbit/jcr/JCRNodeDecorator.groovy

```diff
@@ -105,6 +106,26 @@ class JCRNodeDecorator {
         return definition.isMandatory()
     }
 
+    /**
+     * Returns the "jcr:created", "jcr:lastModified" or "cq:lastModified" date property
+     * for current Jcr Node
+     * Returns null of none of the 3 are found
```

Comment:
We should probably change the doc to something like above. If I had never seen this method before, I would rather the JavaDoc document the intent to help my understanding, rather than the implementation. 

You can even throw something in about delta sync if you think that is helpful too.